tests/vic-machine.bats

```diff
@@ -125,6 +126,10 @@ teardown() {
     run "$docker" "${docker_cmd[@]}" start "$name"
     assert_success
 
+    wget "$vicadmin_ip"/container-logs.tar.gz --no-check-certificate -O install-container-logs.tar.gz
+    wget "$vicadmin_ip"/logs.tar.gz --no-check-certificate -O install-logs.tar.gz
+    assert_success
```

Comment:
Ah, I see. I'll update this. I assume that as it is, the assert_success just does nothing? Would've expected them to fail, heh